projects/train/train/data/base.py

```diff
@@ -264,33 +235,27 @@ def get_slice_bounds(self, total, world_size, rank) -> tuple[int, int]:
         stop = (rank + 1) * per_dev
         return start, stop
 
-    def load_train_signals(self, f, world_size, rank):
+    def load_train_waveforms(self, f, world_size, rank):
```

Comment:
rank is an `int` that corresponds to the GPU you're on for distributed training. `world_size` is the total number of GPUs being used for distributed training. 

Here, we are splitting up the training waveforms amongst GPUs so that each GPU gets unique waveforms.

Reply:
Got it, okay